ext/intl/dateformat/dateformat.stub.php

```diff
@@ -154,10 +154,11 @@ public static function formatObject($datetime, $format = null, ?string $locale =
 
     /**
      * @param int $offset
+     * @param bool $updatecalendar
      * @tentative-return-type
      * @alias datefmt_parse
      */
-    public function parse(string $string, &$offset = null): int|float|false {}
+    public function parse(string $string, &$offset = null, $updatecalendar = false): int|float|false {}
```

Comment:
```suggestion
    public function parse(string $string, &$offset = null, bool $updateCalendar = false): int|float|false {}
```

Shouldn't be camel case in class context?

Reply:
yes true